applications/nrf_desktop/src/hw_interface/Kconfig.passkey

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Passkey input configuration"
+
+choice
+	prompt "Select passkey input interface"
+	default DESKTOP_PASSKEY_NONE
+
+config DESKTOP_PASSKEY_NONE
+	bool "Disable passkey"
+
+config DESKTOP_PASSKEY_BUTTONS
+	bool "Passkey based on button presses"
+	depends on !DESKTOP_BUTTONS_NONE
+
+endchoice
```

Comment:
I don't get it, can't this just be a single boolean option? Why a choice?

Reply:
After changing it to a boolean option, we should merge it into the menu like this:

```
menuconfig M
if M
    config C1
    config C2
endif

```
then we don't need the "depends on !DESKTOP_PASSKEY_NONE"

Reply:
I assumed that we may add various implementations there, but only one of them may be used at a time. This is also consistent with our other modules.

Reply:
If this is likely then this is OK.